include/z64cutscene_commands.h

```diff
@@ -20,8 +20,9 @@
  *   00000001 0001ssss eeee0000
  *   size = 0xC
  */
+#define CS_CAM_EYE_LIST CS_CAM_POS_LIST
 #define CS_CAM_POS_LIST(startFrame, endFrame) \
```

Comment:
I assume the top define is for zapd compatibility?
I think it would make more sense to use the newer name in the main macro, that way in the future only the define with the old name needs removed
This is not critical at all though
```suggestion
#define CS_CAM_POS_LIST CS_CAM_EYE_LIST
#define CS_CAM_EYE_LIST(startFrame, endFrame) \
```